mobile/src/screens/HomeScreen.jsx

```diff
@@ -1,15 +1,40 @@
-import { Button, Text } from "react-native"
+import { Button, FlatList, ScrollView, Text } from "react-native"
 import Page from "../components/Page"
 import screens from "../utils/constants/screens"
+import { getCategories } from "../utils/categoriesServices"
+import { useEffect, useState } from "react"
+import ErrorScreen from "./ErrorScreen"
+import errors from "../utils/constants/errors"
 
 const HomeScreen = props => {
   const { navigation } = props
-  const goToCategory = () => navigation.navigate(screens.categoryPage)
+  const [categories, setCategories] = useState([])
+  const [hadError, setHadError] = useState(false)
+  const goToCategory = id => () => navigation.navigate(screens.categoryPage, { id })
+
+  useEffect(() => {
+    ;(async () => {
+      try {
+        const categoriesResult = await getCategories()
+        setCategories(categoriesResult.data)
+      } catch (e) {
```

Comment:
You catch e here, where is it used ?

Reply:
Yes you at not catching "e", why ?

Reply:
You must catch e if you don't want an error to be thrown. I tried it 
Mb in previous comment I said braquets instead of parentheses

Reply:
But the variable "e", aren't you going to use it ?

Reply:
No has the only thing that can go wrong on this screen is the loading of the categories, if an error is thrown it means  that the category loading failed so no need to use it for the moment. But still need to to catch it to avoid an error. 

But this can be the subject of a rework